pkg/thing/mocks/thing_proxy.go

```diff
@@ -0,0 +1,32 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/thing/delivery/http"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakeThingProxy struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type FakeThingProxy should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)